templates/zerver/help/delete-a-stream.md

```diff
@@ -2,23 +2,14 @@
 
 {!admin-only.md!}
 
-{!subscriptions.md!}
-{!filter-streams.md!}
+{!go-to-streams.md!}
 
-    !!! tip ""
-    {!all-streams.md!}
+{!select-a-stream.md!}
```

Comment:
I added these two templates that are pretty analogous to the old templates. However, they are new templates because I'm trying to do a gradual replacement as I go through the stream documents, rather than trying to do a one-shot thing. The phrasing is a bit awkward, and I would need to go through each usage independently.